app/views/listings/index.html.haml

```diff
@@ -1,3 +1,4 @@
+= render "./partials/navbar"
```

Comment:
The navbar should be included in the application layout

Reply:
done